texts/0006-tauri-releases-schedule.md

```diff
@@ -0,0 +1,53 @@
+- Feature Name: tauri_release_schedule
+- Start Date: 2024-6-6
+- RFC PR: [tauri-apps/rfcs#14](https://github.com/tauri-apps/rfcs/pull/14)
+- Tracking Issue:
+
+# Summary
+
+Define a release schedule for tauri main repo, including crates, NPM packages and CLIs. 
+
+# Motivation
+
+As it stands now, there is no release schedule and releases are made either whenever one of the working group feels like it (mainly Lucas and myself (Amr)),
+or when there is a need for an urgent fix.
+
+As a tauri user or contributor, there is no indication when a release might happen and if will it include a certain feature
+one might be waiting for which leads me to either try to figure how to use unreleased features directly from source or wait indefinitely.   
```

Comment:
What just came to my mind, loosely, I think that the solution is not necessarily how it will be released, but how the code will be built so that it can be easily patched./

Reply:
We do have a F.A.Q about how to use unreleased changes https://tauri.app/v1/guides/faq#how-can-i-use-unpublished-tauri-changes but not all users are willing to use unreleased changes.